### PR TITLE
Random repeater fixes

### PR DESCRIPTION
**Pull request by benrudolph**
@czue @wpride some fixes will brought up in http://manage.dimagi.com/default.asp?218200
:octopus: / :office: 
cc: @NoahCarnahan

### CODE REVIEW

File: corehq/apps/repeaters/views.py [hunk 1]
```diff
@@ -1,8 +1,10 @@
 from django.core.urlresolvers import reverse
 from django.views.generic import View
+from django.http import Http404
```
Comment (dimagimon) [inline corehq/apps/repeaters/views.py]: F401 'Http404' imported but unused

**Comment by wpride** [comment]
:tight:

**Comment by NoahCarnahan** [comment]
:+1:

**Comment by NoahCarnahan** [comment]
(will merge when tests finish)

**Comment by nickpell** [comment]
offlined and am going to merge now and deploy (tests pass locally)

**Comment by czue** [comment]
:+1:

**Pull request merged**